client/gutenberg/extensions/contact-form/editor.js

```diff
@@ -234,7 +234,8 @@ registerBlockType( 'jetpack/field-url', {
 
 registerBlockType( 'jetpack/field-date', {
 	...FieldDefaults,
-	title: __( 'Date' ),
+	title: __( 'Date Picker' ),
+	keywords: [ __( 'Calendar' ), __( 'day month year' ) ],
```

Comment:
> `__( 'day month year' )`

Translators might be confused by this, so adding explanation might help. What do you think of:

> `_x( 'day month year', 'block search term' )`

Reply:
Good call!